README.md

```diff
@@ -1,140 +1,146 @@
+1. Follow the instructions on https://www.raspberrypi.com/software/ to create a new installation of Raspebrry Pi OS.
+2. Connect your raspberry Pi to the internet. 
+3. Create a sudo user and log in. 
+
+## Install docker
+```
+curl -sSL https://get.docker.com | sh
+sudo usermod -aG docker <username>
+```
+
+## Set up influxdb, grafana and udp-to-influx services
+Copy the mira-ruuvitag/RPi directory the the Raspberry Pi.
+```
+scp -r mira-ruuvitag/RPi <username>@<raspberry_pi_ip>:/home/<username>/
+```
+On the Raspberry Pi
+```
+cd /home/<username>/RPi
+docker compose up -d
+```
+
+When running this the first time a influx database will be created where the sensor data will be stored.
+After some time, you should be able to access grafana at <raspberry_pi_ip>:3000 in your browser.
+
+## Install Mira gateway
+Download the latest Mira gateway .deb package installer. It can be found on https://dl.lumenradio.com/mira/. Contact sales@lumenradio.com for more information.
+
+Install the package:
+``` 
+sudo dpkg -i mira-gateway-armhf.deb
+```
+This installs the gateway executable, puts it in a service with autorestart, and reads configuration from /etc/default/mira-gateway and /etc/mira-gateway/mira-gateway.env.
+
+If using MiraUSB, enable the external antenna by uncommenting
+```
+# MIRA_GW_ANTENNA=1
+```
+Enable UDP packet forwarding from gateway to host by uncommenting (only needed if using older version than 2.8.0)
+```
+# MIRA_GW_FORWARD=yes
+```
+in `/etc/default/mira-gateway`.
+
+Start the gateway service and verify that it starts up correctly. Connect a MiraUSB to give the host something to talk to.
+```
+sudo systemctl start mira-gateway
+journalctl -u mira-gateway.service -f
+```
+
+## Verify mira_gateway autostarts
+Time to reboot the raspberry pi to verify that everything is properly setup.
+
+## Setting up RuuviTags
 
 Get the tools required for nRF targets according to:
 https://docs.lumenrad.io/miraos/2.8.0/description/toolchain/tools.html
 
 The software is built to work with MiraOS version 2.2.2 or later.
 
-Download and unpack mira to `src/vendor/libmira`. To get access to libmira contact sales@lumenradio.com.
+Download and unpack mira to `[src vendor libmira](src/vendor/libmira)`. To get access to libmira contact sales@lumenradio.com.
 
 Download and unpack nrf5-sdk, and place in `src/vendor/nrf5-sdk`. nrf5-sdk can be
 [downloaded from Nordic Semiconductor](https://www.nordicsemi.com/Products/Development-software/nrf5-sdk).
-Current supported version is 17.1.0. Note, the
+As of libmira 2.8.0, the supported version is 17.1.0. Note, the
 original directory has a name similar to `nRF5_SDK_17.1.0_ddde560` and has to be
 renamed.
 
+## Bulid
+Stand in the top level directory and run
 ```
 make
 ```
+## Flash
+
+First, add a license to the device on address 0x7F000 of length 0x1000, according to MiraOS documentation:
+
+https://docs.lumenrad.io/miraos/2.8.0/description/licensing/licensing_tool.html#signing-a-connected-device
```

Comment:
I think this is the link that should be used: https://docs.lumenrad.io/miraos/2.8.0/description/licensing/licensing_tool.html#signing-a-device-with-remotely-generated-license

Reply:
goot catch, fixed